Ports - Margaret

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
Just a note:  naming files with spaces can make it harder to run the code from the command-line, and making the filename end with `.rb` will let VS Code highlight syntax properly.

I'd suggest a filename like: `ports_margaret.rb`